### PR TITLE
V2 get command branch

### PR DESCRIPTION
**Pull request by deenliong**
Able to get all module details and view all lesson types in a timetable format if the module is being offered in the semester that the user is planning for

### CODE REVIEW

File: src/main/java/seedu/duke/command/GetModuleCommand.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        this.module = Module.get(moduleCode);
+    }
+
+    @Override
+    public void execute(State state, Ui ui, Storage storage) {
+        // if field is empty, display null in ui
+        ui.addMessage("Module               : " + (module.moduleCode.isEmpty() ? "Nil" : module.moduleCode));
+        ui.addMessage("Module Name          : " + (module.title.isEmpty() ? "Nil" : module.title));
+        ui.addMessage("Module Description   : " + (module.description.isEmpty() ? "Nil" : module.description));
+        ui.addMessage("Module Credit        : " + (module.moduleCredit));
```
Comment (kohnh) [inline src/main/java/seedu/duke/command/GetModuleCommand.java:36]: Can moduleCredit be empty?

File: src/main/java/seedu/duke/command/GetModuleCommand.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        } else {
+            ui.addMessage("Module " + module.moduleCode + " is not offered in this semester"
+                    + ", hence no timetable information is available due to unforseen circumstances");
+        }
+
+        ui.displayUi();
+    }
+
+    // check if module is offered in this semester
+    boolean moduleOfferedInCurrentSem(Module module, State state) {
```
Comment (kohnh) [inline src/main/java/seedu/duke/command/GetModuleCommand.java:69]: maybe can change the name to isOfferedInCurrentSem or IsModuleOfferedInCurrentSem to make it sound more like a boolean function

**Review by kohnh — COMMENTED** [review]
Minor adjustments. Overall looks good.

File: src/main/java/seedu/duke/command/GetModuleCommand.java (hunk 2 above)
Comment (deenliong) [reply src/main/java/seedu/duke/command/GetModuleCommand.java:69]: changed boolean name

File: src/main/java/seedu/duke/command/GetModuleCommand.java (hunk 1 above)
Comment (deenliong) [reply src/main/java/seedu/duke/command/GetModuleCommand.java:36]: added check if module credit is 0

**Review by kohnh — APPROVED** [review]
LGTM

**Pull request merged**